pillar/heroku/micromasters.sls

```diff
@@ -0,0 +1,212 @@
+      'MITXONLINE_CALLBACK_URL': 'https://courses-qa.mitxonline.mit.edu/'
+      'MITXONLINE_URL': 'https://rc.mitxonline.mit.edu/'
+      'OPEN_DISCUSSIONS_API_USERNAME': 'od_mm_rc_api',
+      'OPEN_DISCUSSIONS_BASE_URL': 'https://discussions-rc.odl.mit.edu/',
+      'OPEN_DISCUSSIONS_COOKIE_DOMAIN': 'odl.mit.edu',
+      'OPEN_DISCUSSIONS_COOKIE_NAME': 'discussionsrc',
+      'PGBOUNCER_DEFAULT_POOL_SIZE': 100,
+      'PGBOUNCER_MAX_CLIENT_CONN': 1000,
+      'vault_env_path': 'rc-apps'
+      },
+    'production': {
+      'app_name': 'micromasters-production',
+      'ALLOWED_HOSTS': '["micromasters.mit.edu", "mmfin.mit.edu", "mm.mit.edu"]',
+      'CLOUDFRONT_DIST': 'do5zh7b0lqdye',
+      'EDXORG_BASE_URL': 'https://courses.edx.org',
+      'ELASTICSEARCH_INDEX': 'micromasters',
+      'ELASTICSEARCH_SHARD_COUNT': 2,
+      'env_name': 'production',
+      'FEATURE_EXAMS_CARD_ENABLED': True,
+      'GA_TRACKING_ID': 'UA-5145472-10',
+      'GTM_CONTAINER_ID': 'GTM-NB27R8L',
+      'MAILGUN_FROM_EMAIL': 'no-reply@micromasters.odl.mit.edu',
+      'MAILGUN_URL': 'https://api.mailgun.net/v3/micromasters.odl.mit.edu',
+      'MICROMASTERS_BASE_URL': 'https://micromasters.mit.edu',
+      'MICROMASTERS_CORS_ORIGIN_WHITELIST': "['discussions.odl.mit.edu','odl.mit.edu']",
+      'MICROMASTERS_LOG_LEVEL': 'INFO',
+      'MIDDLEWARE_FEATURE_FLAG_QS_PREFIX': 'XIQ',
+      'MITXONLINE_BASE_URL': 'https://courses.mitxonline.mit.edu'
+      'MITXONLINE_CALLBACK_URL': 'https://courses.mitxonline.mit.edu'
+      'MITXONLINE_URL': 'https://mitxonline.mit.edu/'
+      'OPEN_DISCUSSIONS_API_USERNAME': 'od_mm_prod_api',
+      'OPEN_DISCUSSIONS_BASE_URL': 'https://open.mit.edu',
+      'OPEN_DISCUSSIONS_COOKIE_DOMAIN': 'mit.edu',
+      'OPEN_DISCUSSIONS_COOKIE_NAME': 'discussionsprod',
+      'PGBOUNCER_DEFAULT_POOL_SIZE': 20,
+      'PGBOUNCER_MAX_CLIENT_CONN': 500,
+      'vault_env_path': 'production-apps'
+      }
+} %}
+{% set env_data = env_dict[environment] %}
+{% set business_unit = 'micromasters' %}
+{% set cybersource_creds = salt.vault.read('secret-' ~ business_unit ~ '/cybersource').data %}
+{% set rds_endpoint = salt.boto_rds.get_endpoint('micromasters-{env}-app-db'.format(env=env_data.env_name)) %}
+# Those can be removed once this issue is closed https://github.com/mitodl/micromasters/issues/5314
+{% set exams_audit = salt.vault.read('secret-' ~ business_unit ~ '/exams/audit').data %}
+{% set exams_sftp = salt.vault.read('secret-' ~ business_unit ~ '/exams/sftp').data %}
+{% set exams_sso = salt.vault.read('secret-' ~ business_unit ~ '/exams/sso').data %}
+
+proxy:
+  proxytype: heroku
+
+heroku:
+  app_name: {{ env_data.app_name }}
+  api_key: __vault__::secret-operations/heroku>data>api_key
+  config_vars:
+    ALLOWED_HOSTS: '{{ env_data.ALLOWED_HOSTS|tojson }}'
+    AWS_ACCESS_KEY_ID: __vault__:cache:aws-mitx/creds/read-write-delete-{{ business_unit }}-app-{{ env_data.env_name }}>data>access_key
```

Comment:
This should get updated to match the role defined in https://github.com/mitodl/ol-infrastructure/pull/1380